.github/workflows/prepare-patch-release.yml

```diff
@@ -0,0 +1,71 @@
+name: Prepare patch release
+on:
+  workflow_dispatch:
+
+jobs:
+  prepare-patch-release:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          if [[ ! $GITHUB_REF_NAME =~ ^release/[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            echo this workflow should only be run against long-term release branches
```

Comment:
Having difficulty in visualising what does workflow look like for patch release. Can you help me understand this? Let's say some bug was discovered after 1.11.0 release? What would be the new patch release process?

Reply:
just checking that you've seen the steps in the RELEASING.md under "Preparing a new patch release"? https://github.com/open-telemetry/opentelemetry-python/pull/2649/files#diff-87d6a57a759f2386bf9caf5309b9aa0b9ff0ca70f5e38696e114a1ccdb0e785dR15

Reply:
Ah, I missed that README section. It clarifies my doubt. I'll need to play around with these workflows to get a good grasp.